docs/api/en/textures/Texture.html

```diff
@@ -147,6 +147,13 @@ <h3>[property:number format]</h3>
 			formats.
 		</p>
 
+		<h3>[property:String compareFunc]</h3>
```

Comment:
Since this is only relevant for depth textures, it's best to move this block to the `DepthTexture` page.  

Besides, it seems the PR does not introduce the property in the `DepthTexture` class. It should be defined in the ctor with a proper default value and also honored by copy and serialize/deserialize operations.